### DISCUSSION TITLE
Webhook subscribe to events on other public repository

### DISCUSSION THREAD

**Discussion by RobQuistNL** [community/community#24840] [Apps, API and Webhooks]
<p>I’m working on a package manager for a specific program, and I’m utilizing Github as the host for the packages. People can submit their repositories to add them to the database.</p>
<p>I’ve made a Github App that I’m using from the website now, to fetch information about the repository (tags, contents etc.).</p>
<p>This works great, but I need to keep track of changes in repositories that are listed. Main example is a new tag / release - I need to update my local database with the new version / tag of the package.</p>
<p>Now one idea is to poll the repositories every once in a while and scan for differences - but this feels very inefficient, wasting the ratelimit, bandwidth and resources.</p>
<p>Since the app is private, the webhook only gets activated on changes from my organisation. <strong>Is there a way to subscribe “create” events on other (public) repositories, without the need of the owner installing the app?</strong> This would save a lot of time and resources.</p>
<p>Kind regards,<br>
Rob</p>

**Comment by nyahbhinghiPrincess** [comment, marked as answer]
<p>Hi <a class="mention" href="/robquistnl">@robquistnl</a>,</p>
<p>Thank you for this question!  Your use case sounds very interesting. </p>
<p>Managing (creating/updating/deleting) <a href="https://help.github.com/en/articles/about-webhooks" rel="nofollow noopener">repository webhooks</a> requires admin permissions on that particular repository; so it would not be possible to manage triggering webhooks for public repositories that you do not own.  Having the owners of those repositories install your GitHub app, may be the best approach toward your objective.</p>
